tslint.json

```diff
@@ -44,33 +28,17 @@
       }
     ],
     "no-consecutive-blank-lines": false,
-    "no-console": [
-      true,
-      "debug",
-      "info",
-      "time",
-      "timeEnd",
-      "trace"
-    ],
+    "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
     "no-empty": false,
-    "no-inferrable-types": [
-      true,
-      "ignore-params"
-    ],
+    "no-inferrable-types": [true, "ignore-params"],
     "no-non-null-assertion": true,
     "no-redundant-jsdoc": true,
     "no-switch-case-fall-through": true,
     "no-var-requires": false,
-    "object-literal-key-quotes": [
-      true,
-      "as-needed"
-    ],
+    "object-literal-key-quotes": [false, "as-needed"],
     "object-literal-sort-keys": false,
     "ordered-imports": false,
-    "quotemark": [
-      true,
-      "single"
-    ],
+    "quotemark": [false, "single"],
```

Comment:
i had to use false to disable the ts lint rule here as otherwise this rule was conflicting with prettier. I've done some research on the web and can't find any complications so far using double quotes for strings. 

have done some testing and below passes linting and prettier formatting

```
import { Component } from "@angular/core";

@Component({
  selector: "app-root",
  template: '<div class="some-class-name"></div>',
  styleUrls: ["./app.component.scss"]
})
export class AppComponent {
  public title = "'revalidation'";
  public x = `"'revalidation testsrtestse testsrtestse testsrtestse testsrtestse testsrtestse testsrtestse testsrtestse testsrtestse 
              testsrtestse testsrtestse testsrtestsetestsrtestsetestsrtestsetestsrtestsetestsrtestsetestsrtestse testsrtestse testsrtestse 
              testsrtestse '"`;
}

```

Reply:
Ok